js/components/controlBar.js

```diff
@@ -623,6 +637,36 @@ var ControlBar = React.createClass({
         )
       }).bind(this),
 
+      "multiAudio": (function (alignment) {
+        return (
+          <div className="oo-multiaudio-container" key="multiAudio">
+            <AccessibleButton
+              ref={function(e) { this.toggleButtons[CONSTANTS.MENU_OPTIONS.MULTI_AUDIO] = e }.bind(this)}
+              style={selectedStyle}
+              className={multiAudioClass}
+              focusId={CONSTANTS.FOCUS_IDS.MULTI_AUDIO}
+              ariaLabel={CONSTANTS.ARIA_LABELS.MULTI_AUDIO}
+              ariaHasPopup={true}
+              ariaExpanded={this.props.controller.state.multiAudioOptions.showPopover ? true : null}
+              onClick={this.handleMultiAudioClick}
+            >
+              <Icon {...this.props} icon="multiAudio" style={dynamicStyles.iconCharacter}
+                    onMouseOver={this.highlight} onMouseOut={this.removeHighlight} />
+              <Tooltip enabled={isTooltipEnabled} text={Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.MULTI_AUDIO, this.props.localizableStrings)} responsivenessMultiplier={this.responsiveUIMultiple} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height} alignment={alignment} />
+            </AccessibleButton>
+            {this.props.controller.state.multiAudioOptions.showPopover &&
+            <Popover
+              popoverClassName="oo-popover oo-popover-pull-right"
+              autoFocus={this.props.controller.state.multiAudioOptions.autoFocus}
+              closeActionEnabled={this.props.controller.state.accessibilityControlsEnabled}
+              closeAction={this.closePopover.bind(this, CONSTANTS.MENU_OPTIONS.MULTI_AUDIO)}>
```

Comment:
I think these binds are cleaner than creating a new function specifically for multi audio, and we can keep them, but in general we should avoid these whenever possible because they will be creating a new function on each render.